### PR TITLE
Added additional instructions for contributing: style guide, guidelines, and sca…

### PR DESCRIPTION
**Pull request by cngo-github**
PR Type: Documentation

Added additional instructions for contributing: style guide, guidelines, and _scalafmt_.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,18 @@ git rebase develop
...
    - New/updated tests
 
+# When developing
+
+We roughly follow Scala's styleguide and Cat's guidelines:
+
+- https://docs.scala-lang.org/style/
+- https://typelevel.org/cats/guidelines.html
+
+We also use Scala's formatter, [scalafmt](https://scalameta.org/scalafmt/).
```
Comment (marcinwadon) [inline CONTRIBUTING.md:34]: We use both `scalafmt` and `scalafix`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ git rebase develop
...
+
+## Setting up _scalafmt_
+- For IntelliJ, click [here](https://www.jetbrains.com/help/idea/work-with-scala-formatter.html).
```
Comment (marcinwadon) [inline CONTRIBUTING.md:37]: before commiting, run `sbt runLinter`

**Review by marcinwadon — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (cngo-github) [reply CONTRIBUTING.md:33]: Added.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +24,18 @@ git rebase develop
    - Documentation
    - New/updated tests
 
+# When developing
+
+We roughly follow Scala's styleguide and Cat's guidelines:
```
[the remark this answers is not in the document]
Comment (cngo-github) [reply CONTRIBUTING.md]: Fixed.

[the remark this answers is not in the document]
Comment (cngo-github) [reply CONTRIBUTING.md]: Removed and replace with "Mimic the repository".

File: CONTRIBUTING.md (hunk 1 above)
Comment (cngo-github) [reply CONTRIBUTING.md]: Added `scalafix` link.

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**